src/client/java/teammates/client/scripts/StatisticsPerInstitute.java

```diff
@@ -120,12 +120,10 @@ private StatsBundle generateStatsPerInstitute(
 
             String institute = getInstituteForInstructor(instructor, allAccounts);
 
-            if (!institutes.containsKey(institute)) {
-                institutes.put(institute, new HashMap<Integer, Set<String>>());
-                institutes.get(institute).put(INSTRUCTOR_INDEX, new HashSet<String>());
-                institutes.get(institute).put(STUDENT_INDEX, new HashSet<String>());
-            }
+            institutes.computeIfAbsent(institute, key -> new HashMap<>()).put(INSTRUCTOR_INDEX, new HashSet<>());
+            institutes.get(institute).put(STUDENT_INDEX, new HashSet<>());
             institutes.get(institute).get(INSTRUCTOR_INDEX).add(instructor.getEmail().toLowerCase());
```

Comment:
maybe u want to combine this line with line 123 using `map.computeIfAbsent(key, k -> new HashSet<V>()).add(v);`

Reply:
you mean joining 125 and 123 ?

Reply:
yuppp.
`institutes.computeIfAbsent(institute, key -> new HashMap<>()).put(INSTRUCTOR_INDEX, new HashSet<>());` and `institutes.get(institute).get(INSTRUCTOR_INDEX).add(instructor.getEmail().toLowerCase());`